### ISSUE TITLE
NTLM Authentication

### ISSUE THREAD

**Issue by mikeschoneveld** [Ricorocks-Digital-Agency/Soap#39]
Is it possible to user NTLM authentication with this library?

**Comment by veewee** [comment]
No, it is not possible to perform NTLM authentication with a regular PHP's SOAP client.

You'll need a curl based SoapClient on which you can register `[CURLOPT_HTTPAUTH => CURLAUTH_NTLM, CURLOPT_USERPWD => 'user:pass']`

Linking it to #35.
